nginx-fem-redirects.conf

```diff
@@ -1,4 +1,18 @@
 set $fe_project_uri "https://fe-project.zooniverse.org";
+set $fe_content_pages_uri "https://fe-content-pages.zooniverse.org";
+
+location ~* ^/projects/(?:_next|assets)/.+?$ {
+    resolver 1.1.1.1;
+    proxy_pass $fe_project_uri;
+    proxy_set_header Host fe-project.zooniverse.org;
+}
+
+location ~* ^/about/(?:_next|assets)/.+?$ {
```

Comment:
Should I add the rules for zooniverse.org/about/team and zooniverse.org/about/publications, so that all our URL rewrites re in the same place? That would make more sense, to me, than having those rules in FrontDoor but all the other rewrites here.

Reply:
yes - we should place them here and remove them from front door. 

FWIW - while there is nothing wrong with a regex, I'd be wary of using complex regexes. Instead break them out to multiple but simpler location blocks if needed

Reply:
Good point. I’ll give the page URLs their own block, to keep them separate from the rules that are specific to NextJS config.